spaghetti/network.py

```diff
@@ -3060,7 +3060,6 @@ def element_as_gdf(
     snapped=False,
     routes=None,
     id_col="id",
-    geom_col="geometry",
```

Comment:
Is this a hard breaking change in our public API or not? If so we should probably have some deprecation.

Reply:
Yes, I suppose you are right about this. I will see about adding this in and adjusting the PR.

Reply:
Addressing in [bdd20ee](https://github.com/pysal/spaghetti/pull/724/commits/bdd20eeaeede64a6dc9b34e15811e5543f21cda0)